pytest_reportportal/listener.py

```diff
@@ -58,17 +58,56 @@ def pytest_runtest_makereport(self):
                 # This happens for example when a fixture fails to run
                 # causing the test to error
                 self.result = 'FAILED'
+                self._add_issue_info(item, report)
             elif report.skipped:
                 # This happens when a testcase is marked "skip".  It will
                 # show in reportportal as not requiring investigation.
                 self.result = 'SKIPPED'
-                self.issue['issue_type'] = 'NOT_ISSUE'
+                self._add_issue_info(item, report)
 
         if report.when == 'call':
             if report.passed:
                 item_result = 'PASSED'
             elif report.skipped:
                 item_result = 'SKIPPED'
+                self._add_issue_info(item, report)
             else:
                 item_result = 'FAILED'
+                self._add_issue_info(item, report)
             self.result = item_result
+
+
+    def _add_issue_info(self, item, report):
+
+        issue_type = None
+        comment = ""
+        url = item.session.config.getini('rp_issue_system_url')
+        issue_marks = item.session.config.getini('rp_issue_marks')
+
+        for mark_name in issue_marks:
+            mark = item.get_closest_marker(mark_name)
```

Comment:
this will create regression for folks using pytest 3.4 as `get_closest_marker` is supported from pytest 3.6 onwards.

Reply:
Here is my suggested change in pythonic way. @bigbZik @arozumenko @DzmitryHumianiuk  let me know if this sounds okay. We can as well move it to generic function, because same is required in service.py as well. 
```suggestion
            try:
                mark = item.get_closest_marker(mark_name)
            except AttributeError:
                # pytest < 3.6
                mark = item.get_marker(mark_name)
```